src/components/radioButton.tsx

```diff
@@ -43,17 +43,18 @@ function RadioButton({
   )
 
   return (
-    <label
+    <div
       className={classNames(
-        "block w-12/12",
-        disabled ? "cursor-not-allowed text-grey-4" : "cursor-pointer "
+        "w-12/12",
+        disabled ? "cursor-not-allowed text-grey-4" : "cursor-pointer"
       )}
-      htmlFor={name}
     >
       <WithValidationError error={error}>
         {(hasError) =>
           renderLabel ? (
             <WithHorizontalLabel
+              name={name}
```

Comment:
on naming: it would not be immediately obvious to me that the name I'm passing must match the name of the input. as this is a "label" component, maybe just stick to `htmlFor`, which would maybe allow better to infer the purpose?

it's probably a nitpicky comment. if we only use the `HorizontalLabel` internally, the naming is fine. if we export it, I'd maybe consider giving it another thought

Reply:
Thank you for the review. 🙏 
We use the label component internally currently but I changed `name` for `htmlFor`, it's more meaningful!